test/nft/ZoraCreator1155.t.sol

```diff
@@ -88,12 +88,6 @@ contract ZoraCreator1155Test is Test {
         );
     }
 
-    // NOTE: This won't work on PRs that update the contract version bc package.json is updated via changesets
```

Comment:
i agree with looking into changing this; but that is beyond the scope of this PR and we should come up with a new approach

Reply:
want that back in? it was commented out and never ran on main so it was just cleanup

Reply:
ohhh got it this just deletes the commented out code. 
Sounds good to delete.  Lets review this workflow more long term in the future.